src/widgets/query-rule-custom-data/query-rule-custom-data.tsx

```diff
@@ -13,10 +13,12 @@ import connectQueryRules, {
   QueryRulesRenderState,
   QueryRulesWidgetDescription,
 } from '../../connectors/query-rules/connectQueryRules';
-import CustomData from '../../components/QueryRuleCustomData/QueryRuleCustomData';
+import CustomData, {
+  QueryRuleCustomDataComponentCSSClasses,
+} from '../../components/QueryRuleCustomData/QueryRuleCustomData';
 
 export type QueryRuleCustomDataCSSClasses = {
-  root: string;
+  root?: string;
```

Comment:
```suggestion
  root?: string | string[];
```

Reply:
Fixed in https://github.com/algolia/instantsearch.js/pull/4774/commits/18d2b5ffb91771aecc5905a46d632306769383b4